doc/Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender/index.md

```diff
@@ -0,0 +1,292 @@
+---
+layout: page
+title: Kafka log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender
+sort_idx: 60
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Kafka log appender is responsible for transferring logs from the Operations service to the Apache Kafka service. The logs are stored in the specified topic.
+
+# Creating Kafka log appender in Admin UI
+
+The easiest way to create a Kafka log appender for your application is by using Admin UI.
+
+To create a log appender of the Kafka storage type, do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender<b>.
```

Comment:
< b >Add log appender< b > (missing slash breaks formatting of the rest of the page)